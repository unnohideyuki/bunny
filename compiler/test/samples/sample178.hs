main =do  print $ take 5 [n..]
          print $ take 10 [n,(n-1)..]
          print [1..(m-1)]
          print [1,3..m]
            where n = 9223372036854775807 * 2
                  m = 8

