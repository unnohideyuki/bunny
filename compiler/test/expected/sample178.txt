[18446744073709551614,18446744073709551615,18446744073709551616,18446744073709551617,18446744073709551618]
[18446744073709551614,18446744073709551613,18446744073709551612,18446744073709551611,18446744073709551610,18446744073709551609,18446744073709551608,18446744073709551607,18446744073709551606,18446744073709551605]
[1,2,3,4,5,6,7]
[1,3,5,7]
